Mage.Sets/src/mage/cards/s/SubversiveAcolyte.java

```diff
@@ -0,0 +1,132 @@
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Svyatoslav28
+ */
+public final class SubversiveAcolyte extends CardImpl {
+
+    public SubversiveAcolyte(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{B}");
+
+        this.subtype.add(SubType.HUMAN);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(3);
+
+        // {2}: Pay 2 life: Choose one. Activate only once.
+        // *  Subversive Acolyte becomes a Human Cleric. It gets +1/+1 and gains lifelink.
+        Ability ability = new ActivateOncePerGameActivatedAbility(Zone.BATTLEFIELD, new AddCardSubTypeSourceEffect(Duration.Custom, SubType.HUMAN, SubType.CLERIC).setText("{this} becomes a Human Cleric"),
+                new GenericManaCost(2), TimingRule.INSTANT);
+        ability.addEffect(new BoostSourceEffect(1, 1, Duration.WhileOnBattlefield));
+        ability.addEffect(new GainAbilitySourceEffect(LifelinkAbility.getInstance(), Duration.WhileOnBattlefield));
+        ability.addCost(new PayLifeCost(2));
+
+        // * Subversive Acolyte becomes a Phyrexian. It gets +3/+2 and gains trample and “Whenever this creature is dealt damage, sacrifice that many permanents.”
+        ability.addMode(new Mode(new PhyrexianModeEffect()));
+        this.addAbility(ability);
+    }
+
+    private SubversiveAcolyte(final SubversiveAcolyte card) {
+        super(card);
+    }
+
+    @Override
+    public SubversiveAcolyte copy() {
+        return new SubversiveAcolyte(this);
+    }
+}
+
+class PhyrexianModeEffect extends OneShotEffect {
+
+    PhyrexianModeEffect() {
+        super(Outcome.Benefit);
+        staticText = "Subversive Acolyte becomes a Phyrexian. It gets +3/+2 " +
+                "and gains trample and \"Whenever this creature is dealt damage, sacrifice that many permanents.\"";
+    }
+
+    private PhyrexianModeEffect(final PhyrexianModeEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public PhyrexianModeEffect copy() {
+        return new PhyrexianModeEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent permanent = source.getSourcePermanentIfItStillExists(game);
+        if (permanent == null ) {
+            return false;
+        }
+        game.addEffect(new AddCardSubTypeSourceEffect(Duration.Custom, SubType.PHYREXIAN), source);
+        game.addEffect(new BoostSourceEffect(3, 2, Duration.WhileOnBattlefield), source);
+        game.addEffect(new GainAbilitySourceEffect(TrampleAbility.getInstance(), Duration.WhileOnBattlefield),source);
+        game.addEffect(new GainAbilitySourceEffect(new SubversiveAcolyteTrigerredAbility(), Duration.WhileOnBattlefield),source);
+        return true;
+    }
+}
+class SubversiveAcolyteTrigerredAbility extends TriggeredAbilityImpl {
```

Comment:
looks like this was based on Phyrexian Negator which is overimplemented.

Should be able to use existing `DealtDamageToSourceTriggeredAbility` with `new SacrificeControllerEffect(StaticFilters.FILTER_PERMANENTS, SavedDamageValue.MANY, "")` and no need for custom class